test/jdk/sun/security/pkcs11/ec/ReadCertificates.java

```diff
@@ -55,6 +62,7 @@
 import java.util.LinkedHashMap;
 import java.util.List;
 import java.util.Map;
+
```

Comment:
Please don't make unnecessary changes relative to upstream.

Reply:
Reverted.